src/screens/application/ApplicationStatus.tsx

```diff
@@ -26,14 +12,14 @@ const MyApplications: React.FC = () => {
   };
 
   return (
-    <Layout isNavbar={true}>
+    <Layout
+      isNavbar={true}
+      _heading={{
+        heading: "Pre-matric Scholarship-ST",
```

Comment:
add language constant

Reply:
@sonaliTekdi and I will do this after API integration